flask_resources/loaders.py

```diff
@@ -26,11 +25,12 @@ def wrapper(self, *args, **kwargs):
         """
         # Checking Content-Type is the responsibility of the loader
         loaders = self.resource.config.request_loaders
+        original_request = resource_requestctx.original_request
```

Comment:
Minor: this feels wrong, if flattened the preceding `if`s can be just accessing once the `resource_requestctx`.